README.md

```diff
@@ -1,11 +1,121 @@
-# Stackable Operator for Apache Kafka
+<p align="center">
+  <img width="150" src="./.readme/static/borrowed/Icon_Stackable.svg" alt="Stackable Logo"/>
+</p>
+
+<h1 align="center">Stackable Operator for Apache Kafka</h1>
 
-[![Build Actions Status](https://ci.stackable.tech/job/kafka%2doperator%2dit%2dnightly/badge/icon?subject=Integration%20Tests)](https://ci.stackable.tech/job/kafka%2doperator%2dit%2dnightly)
 [![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://GitHub.com/stackabletech/kafka-operator/graphs/commit-activity)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-green.svg)](https://docs.stackable.tech/home/stable/contributor/index.html)
+[![License OSL3.0](https://img.shields.io/badge/license-OSL3.0-green)](./LICENSE)
+
+[Documentation](https://docs.stackable.tech/kafka-operator/stable/index.html) | [Stackable Data Platform](https://stackable.tech/) | [Platform Docs](https://docs.stackable.tech/)
+
+This is a Kubernetes operator to manage [Apache Kafka](https://kafka.apache.org/) clusters.
+
+It is part of the Stackable Data Platform, a curated selection of the best open source data apps like Kafka, Druid, Trino or Spark, [all](#other-operators) working together seamlessly. Based on Kubernetes, it runs everywhere – [on prem or in the cloud]((#supported-platforms)).
+
+## Installation
+
+You can install the operator using [stackablectl or helm](https://docs.stackable.tech/kafka/stable/getting_started/installation.html).
```

Comment:
I would add a code block here that the user can copy&paste to install the operator.. If the user clicks on the link above, he's not combing back to this readme anymore.

Reply:
If a visitor goes off to the docs and engages with them, that's a win. The job of this README is to invite the reader to trust and try Stackable. Starting to read a more detailed getting-started guide would be an awesome outcome.

I really like the idea of providing an easy on-ramp, and would like to have a "single command, you're good to go" snippet here, but we don't have one. The helm instructions are a bit too verbose for my taste, and `stackablectl` needs to be installed, and it's not as easy as a `curl` or a `brew install`. Also there's the "you probably need a disposable k8s cluster" part, which I fear is not emphasised enough, but that's another topic.

Do you have a snippet in mind which could work here?

Reply:
This is the minimum required to setup a 3 node Kafka cluster. I think it's good to showcase the operator:

```
kubectl apply -f - <<EOF
---
apiVersion: zookeeper.stackable.tech/v1alpha1
kind: ZookeeperCluster
metadata:
  name: simple-zk
spec:
  version: 3.8.0-stackable0.7.1
  servers:
    roleGroups:
      default:
        replicas: 3
---
apiVersion: zookeeper.stackable.tech/v1alpha1
kind: ZookeeperZnode
metadata:
  name: simple-kafka-znode
spec:
  clusterRef:
    name: simple-zk
---
apiVersion: kafka.stackable.tech/v1alpha1
kind: KafkaCluster
metadata:
  name: simple-kafka
spec:
  version: 3.2.0-stackable0.1.0
  zookeeperConfigMapName: simple-kafka-znode
  brokers:
    roleGroups:
      default:
        replicas: 3
EOF
```

Reply:
That would be a good fit for a section below - either "What Does This Do?" or "Getting Started".

At the "Installation" section the reader probably does not have a cluster ready, nor stackablectl, nor any installed operators.

I'm a bit worried to replicate too much of the getting-started style material here, as it will need to be kept up to date, and the new README is better at building confidence already.

Would you be okay to move forward with the current state, and shift the focus towards bringing all operator READMEs to this level of content, 80-20 style? We can iterate on each from there.

Reply:
Yeah sure.